src/shared/components/enhancedReactTable/EnhancedReactTable.tsx

```diff
@@ -165,6 +205,15 @@ export default class EnhancedReactTable extends React.Component<IEnhancedReactTa
         );
     }
 
+    private numPages(itemsPerPage?:number) {
+        itemsPerPage = itemsPerPage || this.state.itemsPerPage;
+        if (itemsPerPage === -1) {
+            return 1;
+        } else {
+            return Math.ceil(this.props.rawData.length / itemsPerPage);
```

Comment:
Actually, we need the length of the "filtered data" here, otherwise we end up with a lot of empty pages when the table is filtered